qemu/tests/usb_device_check.py

```diff
@@ -49,7 +49,16 @@ def _check_test_step_result(result, output):
 
     error_context.context("verify usb devices information in guest...",
                           logging.info)
-    result, output = verify_usb_device_in_guest(params, session, devs)
+    # using dmesg, Linux only
+    result, output = verify_usb_device_in_guest(params, session, devs,
+                                                params["chk_dmesg"])
```

Comment:
`chk_dmesg = "dmesg| grep usb"` this is too general to check the given usb devices, and also I think it is not necessary to verify the usb devices by checking dmesg log.

Reply:
So do you mean I change it back?

Reply:
Yep, thanks.

Reply:
updated thanks